common/plugins/org.polarsys.capella.common.ui.toolkit.browser/src/org/polarsys/capella/common/ui/toolkit/browser/category/CategoryRegistry.java

```diff
@@ -72,7 +72,7 @@ private CategoryRegistry() {
     for (Entry<String, ICategory> entry : entrySet) {
       ICategory category = entry.getValue();
       // top level category and type matching
-      if (category.isTopLevel() && category.isAvailableForType(currentElement)) {
+      if (category.isTopLevel() && category.isAvailableForType(currentElement) && !category.isTechnical()) {
```

Comment:
Can you also a new `public Set<ICategory> gatherTechnicalCategories(String viewerId, EObject currentElement)` to be able to retrieve them

Reply:
Will do, is the approach good for you ?